index.js

```diff
@@ -0,0 +1,7 @@
+const button = document.querySelector('.menu-bars');
+const mobilenav = document.getElementById('mobile-nav');
+button.onclick = function () {
+  mobilenav.style.display = 'block';
+};
+
+console.log('Hello World');
```

Comment:
- [ ]  Congratulations 👏 👏 , You have done a great job implementing the logic of displaying the mobile menu 👍
I think we need also to make the mobile menu disappear when the user click (or taps) on the close(x) button or any option of the mobile menu.

- [ ] I will suggest trying not to leave a console log in your work as this does not follow the best practice of JavaScript.
Kindly check this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) to learn more about JavaScript best practice